clients/extension/scripts/extension-entry.js

```diff
@@ -104,6 +104,8 @@ async function runLighthouseInExtension(flags, categoryIDs) {
   const reportHtml = /** @type {string} */ (runnerResult.report);
   const blobURL = createReportPageAsBlob(reportHtml);
   await new Promise(resolve => chrome.windows.create({url: blobURL}, resolve));
+
+  return runnerResult;
```

Comment:
can you update the return type, too? Since the `undefined` branch throws an error now, `void` isn't necessary any more